webui/react/src/pages/ExperimentList.tsx

```diff
@@ -202,7 +194,8 @@ const ExperimentList: React.FC<Props> = ({ project }) => {
           ...baseParams,
           experimentIdFilter: { notIn: pinnedIds },
           limit: settings.tableLimit - pinnedIds.length,
-          offset: tableOffset - (tableOffset / settings.tableLimit) * pinnedIds.length,
+          offset:
+            settings.tableOffset - (settings.tableOffset / settings.tableLimit) * pinnedIds.length,
```

Comment:
i think this could use some explanatory temporary variables, like.
```
const pageNumber = settings.tableOffset / settings.tableLimit
const rowsTakenUpByPins = pageNumber * pinnedIds.length

```